src/components/search/index.tsx

```diff
@@ -202,6 +228,9 @@ const Search: FunctionComponent<SearchProps> = ({
               <RefinementList attribute="type" />
             </div>
           </div>
+          <div className="justify-center mt-4">
+            <TeamButton />
```

Comment:
this should be conditional. Probably hidden for team owners/members and even pro members. I doubt we have many conversions with pro -> team and no need to "spam" them